app/views/projects/blob/_upload.html.haml

```diff
@@ -0,0 +1,26 @@
+#modal-upload-blob.modal.hide
+  .modal-dialog
+    .modal-content
+      .modal-header
+        %a.close{href: "#", "data-dismiss" => "modal"} ×
+        %h3.page-title Upload 
+        %p.light
+          From branch
+          %strong= @ref
+      .modal-body
+        = form_tag namespace_project_create_blob_path(@project.namespace, @project, @id), method: :post, class: 'form-horizontal', :multipart => true do
+          %br
+          .form-group
+            .col-sm-2
```

Comment:
Remove the empty `.col-sm-2` class and use `.col-sm-offset-2` on the other div.

See https://github.com/gitlabhq/gitlabhq/commit/89bcc1baf040e998730fa7c3e029daf9112321eb